Octokit/Models/Request/CreateFileRequest.cs

```diff
@@ -106,7 +106,7 @@ public class CreateFileRequest : ContentRequest
         /// Creates an instance of a <see cref="CreateFileRequest" />.
         /// </summary>
         /// <param name="message">The message.</param>
-        /// <param name="content">The content.</param>
+        /// <param name="content">The content encoded in base64.</param>
```

Comment:
Yeah, that's a very subtle breaking change. Probably best to use an overload.

Someday I'd like to write a `Base64Content` type. :smile: rather than using strings for everything.